ports/samd/boards/SEEED_WIO_TERMINAL/mpconfigboard.h

```diff
@@ -3,4 +3,11 @@
 
 #define MICROPY_HW_XOSC32K          (1)
 
+#define MICROPY_HW_DEFAULT_UART_ID  (2)
+#define MICROPY_HW_DEFAULT_I2C_ID   (4)
+#define MICROPY_HW_DEFAULT_SPI_ID   (5)
+// Map SDA0/SCL0 to SDA/SCL
+#define pin_SDA  pin_SDA0
+#define pin_SCL  pin_SCL0
```

Comment:
You can put this in pins.csv instead, which I think is a little cleaner, as it keeps all the pin definitions in one place:
```
SDA,PA13
SCL,PA12
```

Reply:
Done.